src/helpers/misc.ts

```diff
@@ -109,6 +111,11 @@ export function getCanonicalSheetName(sheetName: string): string {
   return sheetName;
 }
 
+/** Replace the excel-excluded characters of a sheetName */
+export function sanitizeSheetName(sheetName: string, replacementChar: string = " "): string {
+  return sheetName.replace(sanitizeSheetNameRegex, replacementChar);
```

Comment:
We could use replaceAll rather than re-creating a regex with the global flag

Reply:
the regex is more efficient: https://www.measurethat.net/Benchmarks/Show/2396/0/replaceall-vs-regex-replace#latest_results_block